submissions/evgeniy241984/html-css-popup/styles/index.css

```diff
@@ -0,0 +1,323 @@
+* {
+  box-sizing: border-box;
+}
+
+html {
+  font-family: "Segoe UI", sans-serif;
+  font-size: 13px;
+  color: #686BB1;
+}
+
+body, ul, ol, h1, h2, h3, h4, h5, h6, hr, p, figure, nav, button, input {
+  outline: none;
+  padding: 0;
+  margin: 0;
+}
+
+li {
+  list-style: none;
+}
+
+a {
+  text-decoration: none;
+}
+
+/*main menu*/
+.navigation {
+  padding: 10px 10px 0 0;
+}
+
+.navigation__list {
+    display: -webkit-box;
+    display: -ms-flexbox;
+    display: flex;
+    -webkit-box-orient: horizontal;
+    -webkit-box-direction: normal;
+    -ms-flex-direction: row;
+    flex-direction: row;
+    -webkit-box-pack: end;
+    -ms-flex-pack: end;
+    justify-content: flex-end;
+    -webkit-box-align: center;
+    -ms-flex-align: center;
+    align-items: center;
+}
+
+.list__item {
+  margin-left: 15px;
+  align-items: center;
+}
+
+.list__item_link:hover {
+  text-decoration: underline;
+}
+
+.item__apps {
+  background-image: url("../img/icons8-squared-menu-48.png");
+  height: 30px;
+  width: 30px;
+  background-repeat: no-repeat;
```

Comment:
Try to organize your code logically:  keep `background` rules closer to each other, in the future easier to read your code. Avoid dividing the logic block